### DISCUSSION TITLE
Expansion item group v-model

### DISCUSSION THREAD

**Discussion by labs20** [quasarframework/quasar#11920] [General - Components / Directives / etc] [answered]
Hi. 

Is there a way to have grouped expansion items bounded to a common v-model the same way that tabs and tab panels work?

Thanks!

**Comment by pdanpdan** [comment, marked as answer]
I think you want something like this
https://quasar.dev/vue-components/expansion-item#example--accordion-mode

**Reply by labs20** [reply]
Hi! Thanks for your answer.

I know already how to use the group prop. What I need is a v-model for the entire group, just like tabs has.

```
.my_accordion
    q-expansion-item(group="mygroup" name="acd_1" v-model="active_acordion")
    q-expansion-item(group="mygroup" name="acd_2" v-model="active_acordion")
    q-expansion-item(group="mygroup" name="acd_3" v-model="active_acordion")

...
this.active_acordion = 'acd_2';

```

Thank you.

**Comment by pdanpdan** [comment]
You can try something like this: https://codepen.io/pdanpdan/pen/abLKeRL?editors=1010
It's not directly supported because the expansion item does not have an identifying property (like QTab)

**Reply by labs20** [reply]
Really nice and inspirating code! Thank you. I'll give it a try. 

Thanks!
